x-pack/plugins/security_solution/common/endpoint/schema/actions.ts

```diff
@@ -48,3 +48,23 @@ export const ActionDetailsRequestSchema = {
     action_id: schema.string(),
   }),
 };
+
+export const EndpointActionListRequestSchema = {
+  query: schema.object({
+    agentIds: schema.maybe(
+      schema.arrayOf(schema.string({ minLength: 1 }), { minSize: 1, maxSize: 50 })
+    ),
+    actionTypes: schema.maybe(
+      schema.arrayOf(schema.string({ minLength: 1 }), { minSize: 1, maxSize: 10 })
+    ),
+    page: schema.maybe(schema.number({ defaultValue: 1, min: 1 })),
+    pageSize: schema.maybe(schema.number({ defaultValue: 10, min: 1, max: 100 })),
+    startDate: schema.maybe(schema.string()),
+    endDate: schema.maybe(schema.string()),
+    userIds: schema.maybe(
+      schema.arrayOf(schema.string({ minLength: 1 }), { minSize: 1, maxSize: 20 })
```

Comment:
This is arbitrary. Should we not have any limits on the userids?

Reply:
I think in the past we limited things like this when we needed to query multiple indexes for the values.  I'm ok if you leave it, but it could be an unnecessary limit. BTW - same comment for `actionTypes` (`commands`). 

Example: in the UI, if we ever create search filter drop-down for Commands and Users, we will need to restrict the user to only allow them to pick as many from the list as the api allows - which is probably unnecessary since this is only used as a filter to ES.

Reply:
Makes sense from the UX perspective, and will remove the limits for `commands` and `userIds`.